DEVELOPMENT.md

```diff
@@ -86,6 +86,31 @@ make test-e2e-stash
 | Test organization | go-git-provider-testing      | `GIT_PROVIDER_ORGANIZATION` |
 | Test team         | fluxcd-test-team             | `STASH_TEST_TEAM_NAME`      |
 
+
+#### Gitea
```

Comment:
This section needs mentioning of the `stop-provider-instances` target. Please add it to the GitLab section, too.